modules/Bio/EnsEMBL/Production/Pipeline/GPAD/Base.pm

```diff
@@ -0,0 +1,75 @@
+=head1 LICENSE
```

Comment:
If I've understood the new config correctly, this is redundant now? The cleaning up is done by a combination of AnalysisSetup and RemoveOrphans?

Reply:
Actually, this is still needed. If you re-run the "gpad_file_load" analysis multiple times (e.g: because of a failure) then you will end up with duplicated GOA data... This is to make sure we don't end up in this situation. I could move the deletion back to gpad_file_load module.

Reply:
See latest commit.